edn/src/types.rs

```diff
@@ -24,7 +26,7 @@ pub enum Value {
     Float(OrderedFloat<f64>),
     Text(String),
     Symbol(String),
-    Keyword(String),
+    Keyword(keyword::Keyword),
```

Comment:
Two reasons why not:

- Not all strings are valid keywords.
- Most constructions are namespaced, which makes that approach confusing.